src/components/network-area-diagram-viewer/diagram-utils.ts

```diff
@@ -64,6 +65,224 @@ const EdgeTypeMapping: { [key: string]: EdgeType } = {
 const TEXT_BOX_WIDTH_DEFAULT = 200.0;
 const TEXT_BOX_HEIGHT_DEFAULT = 100.0;
 
+export function getBendableFrom(element: SVGElement): SVGElement | undefined {
+    if (isBendable(element)) {
+        return element;
+    } else if (element.parentElement) {
+        return getBendableFrom(element.parentNode as SVGElement);
+    }
+}
+
+export function isBendable(element: SVGElement): boolean {
+    return hasId(element) && element.parentNode != null && idIs(element.parentNode as SVGElement);
+}
+
+function idIs(element: SVGElement): boolean {
+    return element.id == 'lines-points';
```

Comment:
Rather keep the same principle as other interactive elements, that is, like isSelectable, test the class list.